modules/virt-configuring-a-live-migration-policy.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/live_migration/virt-configuring-live-migration-policies.adoc
+
+
+:_content-type: PROCEDURE
+[id="virt-configuring-a-live-migration-policy_{context}"]
+= Configuring a live migration policy
+
+Use the `MigrationPolicy` custom resource definition (CRD) to define migration policies for one or more groups of selected virtual machines (VMs).
+
+You can specify groups of VMs by using any combination of the following:
+* VM labels such as `size`, `os`, and `gpu`.
+* Namespace labels such as `priority`, `bandwidth`, and `hpc-workload`.
+
+[NOTE]
```

Comment:
I suggest to add a disclaimer that this feature is in `v1alpha` version, therefore APIs might change in the future

Reply:
There will be a TP note on the doc, so this should be implied in general. Leaving this as-is.

Reply:
Added Tech Preview note to docs, which covers this kind of language/disclaimer.